src/components/search-bar.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
File names should have the same name as the component name. 
This could also be turned into a functional component.